### DISCUSSION TITLE
Colors not shown anymore within VS Code

### DISCUSSION THREAD

**Discussion by MartijnHarmenzon** [tailwindlabs/tailwindcss#1775] [Help] [answered]
I believe since Tailwind 1.4.x, colors are not shown anymore by color highlighters/pickers within VS Code. Is anyone else experiencing this issue? Might it have to do with the class bg-opacity? Same is with text colors. I am using the VS Code packages Tailwind CSS IntelliSense and Color Highlight.

![image](https://user-images.githubusercontent.com/23123158/82048017-1e02c880-96b4-11ea-88cc-cd819da087e1.png)

**Comment by LZL0** [comment]
Same.

**Comment by devcircus** [comment, marked as answer]
If you're using the tailwindcss intellisense plugin, it's a known issue that is being worked on. Looks like fix is available in alpha. Not really a tailwind issue. https://github.com/bradlc/vscode-tailwindcss/issues/113

**Reply by MartijnHarmenzon** [reply]
Thanks @devcircus . I can confirm it has been solved with the alpha release of the plugin.
